code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -516,6 +522,35 @@
 		"is stabbing themselves with their mandibles!",
 		"is holding their breath!")
 
+	species_abilities = list(
+		/mob/living/carbon/human/proc/toggle_biolum,
```

Comment:
it does if you insert the organ during surgery, but this is for normally spawned kidans, but maybe I can get rid of it with the action button thing, still have to look into it